src/stories/Textarea.stories.tsx

```diff
@@ -69,7 +69,7 @@ export const TextareaStory = (args: FormControlArgs<TextareaProps>) => {
     <Box as="form" sx={{p: 3}}>
       <FormControl {...parentArgs}>
         <FormControl.Label {...labelArgs} />
```

Comment:
@joshblack the `<FormControl.Label />` is intended to be visually hidden in this example and I remember we talked about it in the last axe-clean call. I didn't fix it here because it is "broken" on some other form elements as well, [TextInput](https://primer.style/react/storybook/?path=/story/components-forms-textinput--default) for example. I am also not sure if we should just fix them to be actually visually hidden or would it make sense to fix it and maybe add another story with labels visible? I am just thinking loudly.

Reply:
Oh wild 🤔 I think you're right that being able to have stories with and without visual labels would be helpful just to demonstrate this (with the default being having a visual label)

Reply:
Reported an issue for this: https://github.com/primer/react/issues/2947